hs628_tools/feature_missing.py

```diff
@@ -0,0 +1,201 @@
+        print('%d columns with a single unique value:\n\n' % len(self.single_unique))
+        for i in range(len(self.single_unique)):
+            print('Colunm # %d with a single value of %s \n' % (single_unique[i], data[1, single_unique[i]]))
+
+        return np.array(self.single_unique)
+
+    def colinear(self, train_data, corr_thresh=0.9):
+        # type: (ndarray, float) -> ndarray
+
+        """
+        Find columns with correlation coefficient value more than specified value
+            -----------
+            Parameters:
+                train_data: numpy ndarray, shape = [n_samples, n_features]
+                      observations and features
+                corr_thresh: float between 0 and 1
+                      Pre-defined threshold set by the user, default value = 0.9
+            -----------
+            Returns:
+                A list of tuples containing a pair of column indices with
+                correlation coefficient value more than specified value
+        """
+
+        # creating a matix for correlation coefficient
+        corr_matrix = np.zeros((train_data.shape[1], train_data.shape[1]))
+
+        for i in range(train_data.shape[1]):
+            for j in range(train_data.shape[1]):
+                # Calculating the corrcoef of each pair and save in corr_matrix
+                corr_matrix[i, j] = np.corrcoef(train_data[:, i], train_data[:, j])[0, 1]
+        # finding pairs with corrcoef more than corr_thresh and save them as a tuple in corr_pairs
+        corr_pairs = []
+        for i in range(corr_matrix.shape[0]):
+            for j in range(i + 1, corr_matrix.shape[0]):
+                if abs(corr_matrix[i, j]) > corr_thresh:
+                    corr_pairs += [(i, j)]
+
+        print('%d features with a correlation greater than %0.2f:\n\n' % (len(corr_pairs), corr_thresh))
+        for i in corr_pairs:
+            print('columns #%d and #%d have a correlation value of %s \n' % (i[0], i[1], corr_matrix[i[0], i[1]]))
+        self.collinear_col = np.array(corr_pairs)
+        return self.collinear_col
+
+    def low_importance(self, features, target, target_type="classification", importance_thresh=0.05):
+        # type: (ndarray, ndarray, str, float) -> ndarray
+
+        """
+        Find columns with their normalized importance value below pre-defined threshold using RandomForestRegressor or
+        RandomForestClassifier based on the target_type.
+            -----------
+            Parameters:
+                features :  numpy ndarrays, , shape = [n_samples, n_features]
+                            Train data, observations and features
+
+                target:     numpy ndarrays, shape = (1, ) target can be binary if target_type is classification or
+                            continuous if target_type is regression
+
+                target_type : string defines the RandomForest method if target is continuous or binary. default value
```

Comment:
I think this comment should mention both possible values for the parameter, not just one, and be a bit more clearly phrased in general.